canonical-websites/www.ubuntu.com
# Bump the global nav to 1.2.1

## Done
Bump the global nav to 1.2.1

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Open the homepage
- Check the console has a message for developers
